src/css/blog-post.module.scss

```diff
@@ -45,6 +46,46 @@
   }
 }
 
+// Index
+
+.indexContainer {
+  display: flex;
+  justify-content: end;
+}
+
+.indexSubContainer {
+  position: fixed;
+  top: 16rem;
+  bottom: 2rem;
+  width: 20rem;
+  margin-right: 6rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 20rem;
+}
+
+.links {
+  @extend .text__index__blueTwo;
+  text-decoration: none;
+  font-family: $primary-font;
+  margin-bottom: 2.5rem;
+  height: 3rem;
+  overflow: hidden;
+}
+
+.selectedLink {
+  font-weight: 900;
```

Comment:
I think this should be in text-styles.css or variavles.css, what do you think?

Reply:
Maybe in variables.scss, because I only need font-weight: 900. So, I think is not okay to create a new text style only for it